query-engine/black-box-tests/tests/metrics/smoke_tests.rs

```diff
@@ -14,6 +15,24 @@ mod smoke_tests {
         schema.to_owned()
     }
 
+    fn assert_value_in_range(metrics: &str, metric: &str, low: f64, high: f64) {
+        let regex = Regex::new(format!(r"{metric}\s+([+-]?\d+(\.\d+)?)").as_str()).unwrap();
```

Comment:
We can also return metrics as Json, that would probably simplify the whole test here :D 
(I know, too late - you already wrote the code)